content/collections/api/en/taxonomy.md

```diff
@@ -9,7 +9,7 @@ eu_endpoint: 'https://analytics.eu.amplitude.com/api/2/'
 postman_link: 'https://www.postman.com/amplitude-dev-docs/workspace/amplitude-developers/folder/20044411-4b8180c4-e06e-478a-9c96-0ccf6cf652e2?action=share&source=copy-link&creator=29131806&ctx=documentation'
 api_status: ga
 lede: |-
-  The Taxonomy API grants Enterprise users the ability to programmatically plan their event schema in the Taxonomy tab.
+  The Taxonomy API grants `Enterprise` users the ability to programmatically plan their event schema in the Taxonomy tab.
```

Comment:
```suggestion
  The Taxonomy API grants Enterprise users the ability to programmatically plan their event schema in the Taxonomy tab.
```

No need to add code formatting to the word Enterprise

Reply:
Some of our customers (who are not on an enterprise plan) kinda missed this part and raised some tickets related to this and I thought highlighting the plan name will at least reduce such instances.